ui/app/AppLayouts/Chat/controls/community/PermissionItem.qml

```diff
@@ -0,0 +1,238 @@
+import QtQuick 2.3
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Utils 0.1
+
+Control{
+    id: root
+
+    implicitHeight: contentItem.implicitHeight
+    implicitWidth: 560 // by design
+
+    property var holdingsListModel: ListModel {}
+    property var permissionsObjectModel: QtObject {}
+    property var channelsListModel: ListModel {}
+    property bool isPrivate: false
+
+    signal editClicked
+    signal duplicateClicked
+    signal removeClicked
+
+    QtObject {
+        id: d
+        readonly property int flowRowHeight: 32
+    }
+    background: Rectangle {
+        border.color: Theme.palette.baseColor2
+        border.width: 1
+        radius: 16
+    }
+
+    contentItem: ColumnLayout {
+        spacing: 0
+
+        Rectangle {
+            id: header
+            color: Theme.palette.baseColor2
+            Layout.fillWidth: true
+            Layout.preferredHeight: 34
+            radius: 16
+
+            RowLayout {
+                anchors.fill: parent
+                spacing: 8
+
+                StatusIcon {
+                    Layout.leftMargin: 19
+                    icon: "checkmark"
+                    Layout.preferredWidth: 11
+                    Layout.preferredHeight: 8
+                }
+
+                StatusBaseText {
+                    Layout.fillWidth: true
+                    text: qsTr("Active")
+                    font.pixelSize: 15
+                }
+
+                StatusIcon {
+                    Layout.rightMargin: 10
+                    visible: root.isPrivate
+                    icon: "hide"
+                    color: Theme.palette.baseColor1
+                }
+            }
+        }
+
+        Flow {
+            id: content
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            Layout.topMargin: 8
+            Layout.rightMargin: 16
+            Layout.leftMargin: 16
+            Layout.bottomMargin: 20
+            spacing: 6
+
+            StatusBaseText {
+                font.pixelSize: 17
+                height: d.flowRowHeight
+                text: qsTr("Anyone who holds")
+                verticalAlignment: Text.AlignVCenter
+            }
+
+            Repeater {
+                model: root.holdingsListModel
+
+                RowLayout {
```

Comment:
You are right. One thing we could then do is treat the operators as a single item and not as a property of an item. I'm not sure if it makes sense in terms of the definition of the model itself but it will solve this possible "issue" you are describing.. 

Ok, I will keep it now like this and will ask to the design team. Thanks to point it out!

Reply:
There are several ways to have line breaks also before connectors:
1. Model with separate entries for connectors
2. Helper proxy model duplicating entries of the model (most generic approach probably)
3. Use a trick to put two items to the flow per single item entry (a bit tricky solution but I have a working solution for that)

Imo it's quite natural to have line breaks in arbitrary place but yes, let's ask design team first.